docs/src/How To Work With Us.doc.js

```diff
@@ -0,0 +1,120 @@
+    Some questions we ask ourselves when prioritizing:
+    How many products/surfaces will benefit?
+    - Ideally, we build things into Gestalt that 2 or more teams need with more teams meaning higher priority.
+    How easy or difficult is it to build?
+    - How many engineering hours will it take to build?
+    - Are there accessibility concerns that should ideally be handled by Gestalt?
+    Is it a dependency to other future work we need to do within Gestalt?
+    - Will this unlock additional functionality that other teams need?
+    We prioritize work following the same cycles as product teams within Pinterest. Knowing other teams needs before prioritization starts will help inform our roadmap. If we do no have capacity for the work, but believe we should add it to Gestalt, we may ask if a product team can continue the work. Otherwise, it will go into our [backlog](https://jira.pinadmin.com/secure/RapidBoard.jspa?rapidView=1936&projectKey=PDS&view=planning.nodetail&issueLimit=100).
+4. **Build or follow along**
+    Our engineering team will pair directly with your engineer if they are the ones who will be taking on the work and help them follow the process below. If the Gestalt team is building the component, we will take on the work and follow the same process. As we develop the documentation, Figma files, and code for the updated or new component, we’d love for you to help review and be part of a final sanity check.`}
+    />
+  </MainSection>,
+);
+
+card(
+  <MainSection name="How can you contribute to Gestalt as an engineer?">
+    <MainSection.Subsection
+      description={`
+We always appreciate the help and contributions of other engineers across Pinterest. It could be new variants, simple bug fixes or building out entire components. Before any code happens though, be sure to follow our [request process](https://paper.dropbox.com/doc/How-to-Work-with-Us--BF7v35d55Z5rEPSQVvkaN8vlAg-KSFgsi7Me5kEqhC7sR2VA#:uid=130985289280146283154377&h2=What-is-the-process-to-request).
+
+1. **Talk to your designer**
+   Checking in with your designers and assure that changes have been approved by the Gestalt team, via the [request process](https://paper.dropbox.com/doc/How-to-Work-with-Us--BF7v35d55Z5rEPSQVvkaN8vlAg-KSFgsi7Me5kEqhC7sR2VA#:uid=130985289280146283154377&h2=What-is-the-process-to-request). We don’t recommend starting a PR on new functionality, no matter how small, without confirming this, as you may spend time on changes that won’t be approved to merge into Gestalt.
```

Comment:
this should be` [request process](#What-is-the-process-to-request-new-additions-or-changes)`

Reply:
Done